modules/logging-upgrading-clo.adoc

```diff
@@ -6,28 +6,100 @@
+----
+
+.. Assign permission to collect and forward application logs by running the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z logging-collector -n openshift-logging
+----
+
+.. Assign permission to collect and forward audit logs by running the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-audit-logs -z logging-collector -n openshift-logging
+----
+
+.. Assign permission to collect and forward infrastructure logs by running the following command:
++
+[source,terminal]
+----
+ $ oc adm policy add-cluster-role-to-user collect-infrastructure-log -z logging-collector -n openshift-logging
+----
++
+////
+. Transfom the current configuration to the new API in Logging 6.
++
+For more information, see link:[Changes to Cluster logging and forwarding in Logging 6].
+////
+
+. Move Vector checkpoints to the new path.
++
+The Vector checkpoints in Logging v5 are located at the path `/var/lib/vector/input*/checkpoints.json`. Move these checkpoints to the path `/var/lib/vector/<namespace>/<clusterlogforwarder cr name>/*`. The following example uses `openshift-logging` as the namespace and `collector` as the ClusterForwarder custom resource name. 
++
+[source,yaml]
+----
+$ ns="openshift-logging"
+$ cr="collector"
+$ for node in $(oc get nodes -o name); do oc debug $node -- chroot /host /bin/bash -c "mkdir -p /var/lib/vector/$ns/$cr" ; done
+$ for node in $(oc get nodes -o name); do oc debug $node -- chroot /host /bin/bash -c "chmod -R 755 /var/lib/vector/$ns" ; done
+$ for node in $(oc get nodes -o name); do echo "### $node ###"; oc debug $node -- chroot /host /bin/bash -c "cp -Ra /var/lib/vector/input* /var/lib/vector/$ns/$cr/"; done
+----
 
-. Select the *openshift-logging* project.
+. Update the {clo} by using the {product-title} web console.
+.. Navigate to *Operators* -> *Installed Operators*.
 
-. Click the *Red Hat OpenShift Logging* Operator.
+.. Select the *openshift-logging* project.
 
-. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.9*, depending on your current update channel.
+.. Click the *Red Hat OpenShift Logging* Operator.
 
-. In the *Change Subscription Update Channel* window, select the latest major version update channel, *stable-5.9*, and click *Save*. Note the `cluster-logging.v5.9.<z>` version.
+.. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link.
 
-. Wait for a few seconds, and then go to *Operators* -> *Installed Operators* to verify that the {clo} version matches the latest `cluster-logging.v5.9.<z>` version.
+.. In the *Change Subscription Update Channel* window, select the latest major version update channel, *stable-6.x*, and click *Save*. Note the `cluster-logging.v6.y.z` version.
```

Comment:
We can not update the latest major version .   the maximum channel is  N+2 version.   For v5.9, the maximum is stable-6.1, For v5.8, the maxinum is stable-6.0.

Reply:
Updated the text and added a message about N+2 version